crates/core/tedge_agent/src/operation_workflows/tests.rs

```diff
@@ -204,13 +204,13 @@ async fn convert_outgoing_software_list_response() -> Result<(), DynError> {
         SoftwareListCommand::new(&EntityTopicId::default_main_device(), "1234".to_string());
     let software_list_response = software_list_request
         .clone()
-        .with_status(CommandStatus::Executing);
+        .with_status(CommandStatus::Successful);
     software_box.send(software_list_response.into()).await?;
 
     mqtt_box
         .assert_received([MqttMessage::new(
             &Topic::new_unchecked("te/device/main///cmd/software_list/1234"),
-            r#"{"status":"executing"}"#,
+            r#"{"status":"successful"}"#,
```

Comment:
3 tests has been updated, watching for a *successful* status and no more an *executing* status.

Indeed, publishing the *executing* state is done by the workflow engine according to the workflow definition and no more by the internal actor.